map_server/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
+---
+bool success
```

Comment:
I can submit a PR to common_msgs/nav_msgs for LoadMap.srv. Do you think the service would pass scrutiny in that repo?

Would that mean a path for this change to be released is to wait for the service to be released in common_msgs, then modify this PR to not contain the LoadMap service?